src/test/java/se/bjurr/violations/comments/lib/CommentsCreatorTest.java

```diff
@@ -83,8 +83,8 @@ public boolean shouldKeepOldComments() {
   private List<ChangedFile> files;
   private Integer maxCommentSize;
   private List<Comment> removeComments;
-  private boolean shouldCreateCommentWithAllSingleFileComments = true;
```

Comment:
I agree that these can be made clearer by renaming.

The problem with renaming is that it should then be renamed everywhere. In all maven, gradle and jenkins plugins.... The problem with that is that a lot of Jenkinsfiles, Gradle and Maven build scripts will need to be changed.

Reply:
Ok. If you can tell me where I have to change it, I will do it, since we agree that it makes it clearer.
Is it also the reason for the build failure?

Reply:
The Maven, Gradle and Jenkins plugins are listed in the readme here: https://github.com/tomasbjerre/violations-lib

It needs to be backwards compatible so that all the thousands of users don't need to update there build scripts with this new version. The new parameters can be mentioned in docs while the old parameters are not. But still the old parameters need to have same effect as before.